pkgs/by-name/ve/vesktop/package.nix

```
@@ -16,6 +16,7 @@
   pnpm,
   nodejs,
   withTTS ? true,
+  withMiddleClickScroll ? false,
```

```suggestion
  withMiddleClickScroll ? true,
```
not much reason to leave this out by default

---

This disables middle click pasting from what I can see, and I'm sure some people rely on that

---

ah TIL. thanks!